openhtf/util/measurements.py

```diff
@@ -93,16 +95,23 @@ class MultipleValidatorsException(Exception):
   """Multiple validators were used when defining a measurement."""
 
 
+class Outcome(Enum):
+  """Enum values for measurement 'outcome' fields."""
+  PASS = 'PASS'
```

Comment:
Why do you set the value to its string version? `value.name` returns the string version when you need it.

```py
Outcome = enum.Enum('Outcome', ['PASS', 'FAIL', 'UNSET'])